### DISCUSSION TITLE
file menu on disk insert

### DISCUSSION THREAD

**Discussion by xpertvis** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2861] [General]
Hello,
Using TFT 35 V3.0 LCD.
is there an option, in config file or somewhere, to enable automatic open of file menu, when USB or SD card is inserted ?
I'm unable to find this.
I'm able to configure my own formware.
Thanks

**Comment by kisslorand** [comment]
No, there's no such option implemented.
